react/components/Iframe.js

```diff
@@ -149,8 +160,8 @@ class Iframe extends Component {
     const src = this.fixedSrc
     return loaded ? (
       <iframe
-        className="w-100 calc--height overflow-container"
```

Comment:
Is not a problem, but I didn't understand why remove `w-100` and `overflow-container` from here too.

Reply:
We need fine grained control over the height gap, and unfortunately tachyons can't provide that. I considered leaving the classNames and overriding the `calc--height` style through the css prop, but it seemed better to move every style to the css prop instead of having a mix of both (that would look inconsistent).